aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -49,7 +49,9 @@ Consider using a NameValueList for the following use cases:
 
 ## Anatomy of a NameValuePair
 
+<Example caption="Example showing diagram of NameValueList." plain>
```

Comment:
```suggestion
<Example caption="Diagram illustrating the four parts composing a name-value pair." plain>
```

Reply:
Small recommendation -- where do we want to lean in terms of `name-value pair` vs NameValuePair?